packages/react/src/filter-sidebar/docs/overview.mdx

```diff
@@ -8,7 +8,7 @@ relatedPatterns: ['search-filters']
 
 Filter sidebar is used in catalogue or search results pages to contain filtering options for a list of Cards.
 
-The Filter sidebar collapses on mobile devices, and displays a number of active filter options in the header.
+The Filter sidebar collapses on mobile devices into an accordion, where the number of active filter options is displayed prominently in the header. Individual filter tags are displayed outside of the accordion, providing users with a clear, interactive way to view and remove specific filters as needed.
```

Comment:
Filter tags aren't part of this component, so I think this should be more "if you add filter tags" or it should be added to the patterns/search-filters documentation